shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/plants/presentation/detail/PlantDetailComponent.kt

```diff
@@ -31,7 +37,12 @@ class PlantDetailComponent(
 
     fun onEvent(event: PlantDetailEvent) {
         when (event) {
-            PlantDetailEvent.CheckoutPlant -> TODO()
+            PlantDetailEvent.CheckoutPlant -> {
+                componentCoroutineScope().launch {
+                    basketRepository.insertItem(state.value.plant.id, state.value.quantity)
+                }
+                onShowMessage(UiText.StringRes(MR.strings.plant_added_text, listOf(state.value.plant.name)))
```

Comment:
After adding an item to the basket, clicking the checkout button again may seem like nothing happened (with no additional message), even if more items have been added to the basket. It would be nice to provide some sort of indication when more items are added to the basket :)

Reply:
I added a new logic here :) Maybe it can prevent this nothing happened feeling